Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/ViewBase.cs

```diff
@@ -111,6 +112,12 @@ protected void CreateSelector(Func<InputActionsEditorState, TViewState> selector
         private IViewStateSelector<TViewState> m_ViewStateSelector;
         private IList<IView> m_ChildViews;
         private bool m_IsFirstUpdate = true;
+
+#if UNITY_STANDALONE_OSX
```

Comment:
I believe this is a bit misleading name since it also evaluates if D is pressed. Name suggests only command or control is pressed.

Maybe rename to `IsDuplicateShortcutCommand` if implemented this way?
Also consider having the #ifdef on only the lambda part and not the function signature since they are equivalent? 

Alternatively use `bool isCommandOrControlPressed(KeyDownEvent) => return keyDownEvent.modifiers == EventModifiers.Command;` and check KeyCode similar to other commands for consistency, or make them all evaluated as bool conditions in a common place?

Reply:
Good Points! I updated the code accordingly